client/my-sites/checkout/composite-checkout/components/checkout-loading-placeholder.tsx

```diff
@@ -0,0 +1,63 @@
+import { LoadingContent } from '@automattic/composite-checkout';
+import { styled } from '@automattic/wpcom-checkout';
+import { useTranslate } from 'i18n-calypso';
+import { useRef } from 'react';
+import { CALYPSO_CONTACT } from 'calypso/lib/url/support';
+
+export function CheckoutLoadingPlaceholder( {
+	checkoutLoadingConditions,
+}: {
+	checkoutLoadingConditions: Array< { name: string; isLoading: boolean } >;
+} ) {
+	const initialRenderTime = useRef( Date.now() );
+	const showLoadingInfoThresholdMs = 1000;
```

Comment:
I know they're not measuring the exact same point, but I think we should bump this to just above the average load time from our RUM logs - `4000` looks like it would clear for most people. Is that too delayed to be helpful for tests?